### ISSUE TITLE
default timeout on https requests?

### ISSUE THREAD

**Issue by brute-force** [micnews/apple-news#21]
i couldn't find any definitive info on timeout settings - it seems like it just uses the default OS socket timeout?

**Comment by ellell** [comment]
@brute-force yes, there's no request timeout set explicitly currently. Happy to take a PR for enabling setting this through setup options!

**Comment by brute-force** [comment]
@ellell bit of a wait but done
